### PR TITLE
PMM-11128: nodes info v2

### PR DESCRIPTION
**Pull request by ritbl**
[PMM-11128](https://jira.percona.com/browse/PMM-10541)

Build: 
- https://github.com/Percona-Lab/pmm-submodules/pull/2920

### CODE REVIEW

File: managed/services/telemetry/transform.go [hunk 1]
```diff
@@ -41,7 +41,8 @@ func transformToJSON(config *Config, metrics []*pmmv1.ServerMetric_Metric) ([]*p
 		return nil, errors.Errorf("invalid metrics config")
 	}
 
-	firstMetric := config.Data[0].MetricName
+	// consider first metric is the beginning of the object window
+	firstMetric := metrics[0].Key
```
Comment (ritbl) [inline managed/services/telemetry/transform.go:45]: small bug, that was found while testing

**Review by 0leksii — APPROVED** [review]

File: managed/services/telemetry/transform.go (hunk 1 above)
Comment (gen1us2k) [reply managed/services/telemetry/transform.go:45]: There can be panic once there are no metrics returned

**Review by gen1us2k — APPROVED** [review]

File: managed/services/telemetry/transform.go (hunk 1 above)
Comment (ritbl) [reply managed/services/telemetry/transform.go:45]: if metrics cannot be collected, I think, we should not panic.

without this change if would with "return nil, errors.Errorf("invalid metrics sequence: no match with first metric")", which will be logged.

**Review by atymchuk — APPROVED** [review]

**Comment by atymchuk** [comment]
@ritbl Looks like we need to fix the API test `Test_transformToJSON`

File: managed/services/telemetry/transform.go (hunk 1 above)
Comment (ritbl) [reply managed/services/telemetry/transform.go:45]: @gen1us2k I see what you do mean, I will fix that potential case and write test for it

File: docker-compose.yml [hunk 2]
```diff
@@ -21,23 +21,24 @@ services:
...
     extra_hosts:
       - host.docker.internal:host-gateway
-    #      - portal.localhost:${PORTAL_HOST:-host-gateway}
-    #      - check.localhost:${PORTAL_CHECK_HOST:-host-gateway}
-    #      - pmm.localhost:${PORTAL_PMM_HOST:-host-gateway}
-    #      - check-dev.percona.com:${PORTAL_PMM_HOST:-host-gateway}
+    #  - portal.localhost:${PORTAL_HOST:-host-gateway}
+    #  - check.localhost:${PORTAL_CHECK_HOST:-host-gateway}
+    #  - pmm.localhost:${PORTAL_PMM_HOST:-host-gateway}
+    #  - check-dev.percona.com:${PORTAL_PMM_HOST:-host-gateway}
```
Comment (pkadej) [inline docker-compose.yml:41]: I think we can unstage whitespaces/comments formatting.

**Review by pkadej — APPROVED** [review]

**Review by pkadej — APPROVED** [review]

File: managed/services/telemetry/transform.go (hunk 1 above)
Comment (gen1us2k) [reply managed/services/telemetry/transform.go:45]: I didn't see `if metrics == nil` somewhere above the code.

**Pull request merged**